packages/eslint-plugin/test/rules/jsx-import.test.ts

```diff
@@ -233,18 +233,17 @@ let ele = <div css={{}} />
       code: `
 /** @jsx jsx */
 import * as emotion from '@emotion/react'
-let ele = <div css={\`color:hotpink;\`} />
+let ele = <div css={{}} />
       `.trim(),
       errors: [
         {
           messageId: 'cssPropWithPragma'
         }
       ],
       output: `
-/** @jsx jsx */
 /** @jsx emotion.jsx */
 import * as emotion from '@emotion/react'
-let ele = <div css={\`color:hotpink;\`} />
```

Comment:
With TypeScript-ESLint 8's RuleTester, this causes an error due to overlapping rules for `cssPropWithPragma` and `templateLiterals`. The new RuleTester tries to apply fixes for all errors, not just the first encountered.

To fix this, I split it into two tests, one for each.